operator/docs/standard-operational-procedures/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Standard Operational Procedures"
+description: "The LokiStack Alerts and Standard Operational Procedures"
+lead: ""
+date: 2022-06-21T08:48:45+00:00
+lastmod: 2022-06-21T08:48:45+00:00
+draft: false
+images: []
+---
+
+The following page describes Standard Operational Procedures for alerts provided and managed by the Loki Operator for any LokiStack instance.
```

Comment:
Could you add a sentence or two here to explain Standard Operating Procedures **to do what**?  Resolve the alerts?  Are these pages describing automated actions that the Operator is taking to manage these alerts?  Or are these manual steps that the user has to take?